src/Mod/Draft/DraftEdit.py

```diff
@@ -26,23 +26,24 @@
                 Dmitry Chigrin, Carlo Pavan"
 __url__ = "https://www.freecadweb.org"
 
-import FreeCAD
-import Draft
+import FreeCAD as App
 import math
 
+from Draft import get3DView, getType
 from FreeCAD import Vector
 
-# Do not import GUI-related modules if GUI is not there
-if FreeCAD.GuiUp:
-    import FreeCADGui
+if App.GuiUp:
+    # Do not import GUI-related modules if GUI is not there
```

Comment:
By the way, I don't quite understand this test. As far as I can tell, `Draft_Edit` is already a GUI Command, therefore, we should assume the interface is up. Is there a point in defining it without the interface?

I'm not saying this is wrong, I'm just thinking this can be improved. I realize that when I wrote the PolarArray and CircularArray commands I sort of did the same, copying previous code. However, I do want to revisit this at another time.

Reply:
Yup. I agree, I had the same feeling. 
If also Yorik agree no problem to get rid of it.

Reply:
@vocx-fc.... I've been thinking about it: 
atm Draft_Edit it's contains also lot of code to modify objects: I can use for example update(obj, index, Vector) to arbitrary move a point of a given object.
And it's the only tool we have to modify DraftBezcurve vertexes and continuity.
When we will split all that code, the command will be 100% Gui, and then we can discard this check at the beginning.

Reply:
> Yup. I agree, I had the same feeling.
> If also Yorik agree no problem to get rid of it.

Sure!! What @vocx-fc says seems right indeed.

Reply:
Ok. what about discarding it in next PR? We have already had a huge revisions list in here :)